vacasa/connect/connect.py

```diff
@@ -557,14 +557,14 @@ def get_unit_amenity_properties(self, params: dict = None):
     def add_unit_amenity_property(self,
                                   amenity_id: int,
                                   unit_id: int,
-                                  idamenities_property: int,
+                                  idamenities_properties: int,
                                   property_value: str) -> dict:
         """Add a new unit amenity property in Connect.
 
         Args:
             amenity_id: An amenity ID.
             unit_id: A unit ID.
-            idamenities_property: An amenity property ID.
+            idamenities_properties: An amenity property ID.
```

Comment:
This is still a single integer even though the name is now plural?

Reply:
Would `amenity_property_id` be more consistent with the other field names?

Reply:
I'm fine with the name. Just wanted to make sure the type hint is still accurate.

Reply:
Per the connect API it should be `idamenities_property` why do we need this change?

Reply:
`idamenities_properties` returns successful, `idamenities_property` throws a 400 error.  In addition to this code change, the documentation should be updated to reflect API behavior.

Reply:
Yes, it's a single integer.  I'd guess there was a conflict to call it `idamenities_properties` instead of `amenity_property_id`.

Reply:
Oh I see, you're just matching the Connect API name. 👍